watchdog/src/config.rs

```diff
@@ -78,6 +90,8 @@ impl Config {
             blocks_behind_threshold: BLOCKS_BEHIND_THRESHOLD,
             blocks_ahead_threshold: BLOCKS_AHEAD_THRESHOLD,
             min_explorers: MIN_EXPLORERS,
+            bitcoin_canister_principal: Principal::from_text(MAINNET_BITCOIN_CANISTER_PRINCIPAL)
```

Comment:
Shouldn't the endpoint below be a function of the canister principal?

Reply:
Initially I thought of that too. But I suppose that `<principal>.raw.ic0.app` is also a temporary value and may change some time in the future. Therefore the simplest solution I found is to have two constants with principal and a test to verify that principals are the same.

Reply:
Done.